janus_core/calculations/md.py

```diff
@@ -499,6 +509,80 @@ def __init__(
 
         self._parse_correlations()
 
+    def _set_time_step(self):
+        """Set time in fs and current dynamics step to info."""
+        time = (self.offset * self.timestep + self.dyn.get_time()) / units.fs
+        step = self.offset + self.dyn.nsteps
+        self.dyn.atoms.info["time_fs"] = time
+        self.dyn.atoms.info["step"] = step
+
+    def _prepare_restart(self) -> None:
+        """Prepare restart files, structure and offset."""
+        # Check offset can be read from steps
+        try:
+            with open(self.stats_file, encoding="utf8") as stats_file:
+                last_line = stats_file.readlines()[-1].split()
```

Comment:
Good point.

Since this is just maintaining the [current behaviour](https://github.com/stfc/janus-core/blob/main/janus_core/calculations/md.py#L907), I'll open this as a separate issue